executor/delete.go

```diff
@@ -165,10 +171,16 @@ func (e *DeleteExec) removeRowsInTblRowMap(tblRowMap tableRowMapType) error {
 }
 
 func (e *DeleteExec) removeRow(ctx sessionctx.Context, t table.Table, h int64, data []types.Datum) error {
-	err := t.RemoveRecord(ctx, h, data)
+	// err is always nil if `active` is false.
+	txnState, err := e.ctx.Txn(false)
+	if err != nil {
+	}
+	memUsageOfTxnState := txnState.Size()
+	err = t.RemoveRecord(ctx, h, data)
```

Comment:
This is only one line change, is it easier to estimate the size of data?

Reply:
Using the existing interface is easier?